PagerDuty-Services-Integration/example_inputs/inputs_1_create.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Type": "event_transformer_api_inbound_integration",
+    "Name":"Test Integration For Service",
+    "ServiceId": "P4KW27I",
```

Comment:
I think this should be a place holder so it is clear it needs updated

Reply:
done